.example.env

```diff
@@ -1 +1,3 @@
-DATABASE_URL='mongodb+srv://ridasbnesalmas:rida123@ridacluster.nixyljy.mongodb.net/tourTrek'
\ No newline at end of file
+DATABASE_URL='mongodb+srv://ridasbnesalmas:rida123@ridacluster.nixyljy.mongodb.net/tourTrek'
+NEXTAUTH_SECRET='NEXTAUTH_SECRET'
+NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME='dh92safyd'
```

Comment:
hey Fatimzahra good work on implementing the side bar feature , however it is not a good practice to leave the env credentials shown public like that , overall it is a good job.